airflow/www/views.py

```diff
@@ -2653,3 +2654,11 @@ def get_count_query(self):
             .filter(models.DagModel.is_active)
             .filter(~models.DagModel.is_subdag)
         )
+
+
+class TestAPI(MethodView):
```

Comment:
This is not an API in the context of Airflow, furthermore I don't see the need to include a TestAPI into production